package.json

```diff
@@ -96,6 +96,7 @@
     "resource-router-middleware": "^0.6.0",
     "sharp": "^0.23.4",
     "soap": "^0.25.0",
+    "storefront-query-builder": "https://github.com/DivanteLtd/storefront-query-builder.git",
```

Comment:
I needed to update this to 0.0.9, because yarn keeps version 0.0.8. So right now there is error
```
(node:12110) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'buildQueryBodyFromSearchQuery' of undefined
```